arcgis-ios-sdk-samples/Geometry/Buffer/README.md

```diff
@@ -1,26 +1,39 @@
 # Buffer
 
-This sample demonstrates how to use `AGSGeometryEngine` to create polygons from a map location and linear distance (radius). For each input location, the sample creates two buffer polygons (using the same distance) and displays them on the map using different symbols. One polygon is calculated using the `planar` (projected) coordinate space of the map. The other is created using a `geodesic` technique that considers the curved shape of the Earth's surface. Distortion in the map increases as you move away from the standard parallels of the spatial reference's projection. This map uses the Web Mercator projection so areas near the equator are the most accurate. As you move the buffer location north or south from that latitude, you'll see a greater difference in the polygon size and shape. Planar operations are generally faster, but performance improvement may only be noticeable for large or complex buffer operations.
+Create a buffer around a map point and display the results as an `AGSGraphic`.
```

Comment:
You updated the description in `ContentPList.plist`, but not here. They should match.

```suggestion
Create a buffer around a map point and display the results as a graphic.
```

Reply:
🤦‍♂ Will change!